doc/source/basics.rst

```diff
@@ -220,47 +240,38 @@ received bytes as Commands and to ensure that incoming and outgoing bytes abide
 by the protocol.
 
 .. ipython:: python
+    
+    class OurQueue:
```

Comment:
Do we need to use a custom queue class for this initial example, or with the default queue do?